data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,535 @@
+			label questions
+			choice
+				`	"Could I do anything to help?"`
+					goto help
+					to display
+						not "Hai Reveal Concern Period: Raid on Allhome (local): label help"
+				`	"Who are the Unfettered?"`
+					goto unfettered
+					to display
+						not "First Contact: Unfettered: offered"
+						not "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
+				`	"Thank you for your time."`
+					goto end
+					to display
+						or
+							has "Hai Reveal Concern Period: Raid on Allhome (local): label help"
+							has "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
+			label unfettered
+			action
+				set "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
+			`	She seems surprised that you asked and responds, "Oh, they're a sort of rebel Hai group. If you don't know you could go visit Cloudfire in the Wah Ki system up north, or maybe even Darkcloak if you dare. A short visit would explain it far better than I ever could.`
+			`	"Be very careful if you do though, they're very likely to shoot at you."`
+				goto questions
+			label help
+			action
+				set "Hai Reveal Concern Period: Raid on Allhome (local): label help"
+			`	She shrugs. "I've heard talk of some things that have been suggested to help, but I don't know myself. I guess you should just do the right thing, whatever that is."`
+			`	She seems to be at a loss for what else to say.`
+				goto questions
+			label end
+			`	"No, thank you for caring," she says. "Whatever you do, just be cautious about it. People are scared, and I have a hunch that when they're under pressure, the Hai are a little stranger than people think they are.`
+			`	"Safe travels, Captain."`
+				decline
+
+
+mission "Hai Reveal Concern Period: Raid on Allhome (away)"
+	landing
+	invisible
+	source
+		government "Hai"
+		not planet "Allhome"
+		attributes "hai: human presence"
+		not attributes "uninhabited" "retirement"
+	to offer
+		has "event: hr: pirates respond to leaks"
+		has "First Contact: Unfettered: offered"
+		not "Hai Reveal Concern Period: Raid on Allhome (local): offered"
+		random < 20
+	on offer
+		set "hr: allhome raided"
+		conversation
+			`A wall screen situated between two food stands just outside the docking area is playing through a news segment, and your ears suddenly perk up at the word "marauder".`
+			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew hijacked a vessel with some of the crew still on board and made off with it successfully. At this time it is unclear what has happened to those crewhai."`
```

Comment:
The Hai version of "crewpersons"

Reply:
This isn't immediately obvious, and I hadn't heard of the word 'crewpersons' before. It could stay, but I feel like that would lead on to a few PRs in the future trying to correct it.

Reply:
It wouldn't be the first thing to do so.

Reply:
That was my immediate reaction too, I think there should be an option to ask the Hai what the broadcast meant by “crewhai.”